js/app.js

```diff
@@ -0,0 +1,25 @@
+window.addEventListener("scroll", function () {
+  const fixedHeader = document.querySelector(".header");

```

Comment:
```suggestion
  const header = document.querySelector(".header");

```
まだこの時点ではfixedではないですよね

Reply:
fixedHeader を header に変更します。